### PR TITLE
Add Java Course

### PR DESCRIPTION
**Pull request by ayushhkkkk3585**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
This PR is to add a Java course for the beginners from Youtube to the course section.

### Why is this valuable (or not)?
It is one of the best courses for beginners to learn about Java.

### How do we know it's really free?
It is a free playlist available on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It is video course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -830,6 +830,7 @@
 * [Introduction to Java](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-092-introduction-to-programming-in-java-january-iap-2010/)
 * [Introduction to Java + DSA](https://youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ) - Kunal Kushwaha
 * [Java Beginner](https://youtube.com/playlist?list=PLkeaG1zpPTHiMjczpmZ6ALd46VjjiQJ_8) - Coding with John
+* [Java Complete Course \| Placement Series](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3LtFWcvwpqTkUSlB32kJop) - Apna College
```
Comment (sohansai) [inline courses/free-courses-en.md:833]:
```suggestion
* [Java Complete Course - Placement Series](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3LtFWcvwpqTkUSlB32kJop) - Apna College
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by morhadi** [comment]
how do you put that tick in checklist

**Comment by ayushhkkkk3585** [comment]
> how do you put that tick in checklist
While creating PR, in the boxes besides conditions put a 'x' mark and it will automatically get converted into checklist. You can also verify this by looking it into preview section.

**Pull request merged**